lib/reporters/xunit.js

```diff
@@ -155,7 +155,8 @@ XUnit.prototype.test = function (test) {
     var err = test.err;
     this.write(tag('testcase', attrs, false, tag('failure', {}, false, escape(err.message) + '\n' + escape(err.stack))));
   } else if (test.isPending()) {
-    this.write(tag('testcase', attrs, false, tag('skipped', {}, true)));
+    var reasonTag = test.reason ? tag('reason', {}, false, '<![CDATA[' + test.reason + ']]>') : '';
```

Comment:
I'm going to assume "reason" is a real thing per the XUnit spec?  (I don't use XUnit, so I don't know)

Reply:
@boneskull yes, in XUnit.net `[Fact(Skip="reason")]` or ` [Ignore("reason")]` are used to skip a `Fact/Test` and provide a reason for skipping.

Reply:
@Bamieh happy to have someone around that actually uses XUnit :smile: